src/main/java/reposense/util/FileUtil.java

```diff
@@ -41,6 +43,52 @@
 
     private static final String MESSAGE_INVALID_FILE_PATH = "\"%s\" is an invalid file path. Skipping this directory.";
 
+    /**
+     * Returns a list of paths to the generated reports from {@code repoConfigs}.
```

Comment:
Do include `sourcePath` in your description. The comment is not really describing what it is doing.

In fact, it is possible to do without this method. Since these path were created in `ReportGenerator.generateReposReport`, you can simply tweak it to return the paths.

Reply:
Integrated the method into `ReportGenerator#generateReposReport`.